backend/app/Http/Controllers/API/FavoriteController.php

```
@@ -0,0 +1,189 @@
+    public function store(Request $request): JsonResponse
+    {
+        try {
+            $user = Auth::user();
+
+            // バリデーション
+            $validator = Validator::make($request->all(), [
+                'recipe_id' => [
+                    'required',
+                    'integer',
+                    'min:1',
+                    function ($attribute, $value, $fail) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attribute'. (Unused Code Rules)

(UnusedFormalParameter)